src/include/binder/bound_comment_on.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "bound_statement_result.h"
+#include "common/statement_type.h"
+
+namespace kuzu {
+namespace binder {
+
+class BoundCommentOn : public BoundStatement {
+public:
+    BoundCommentOn(common::table_id_t tableID, std::string tableName, std::string comment)
+        : BoundStatement{common::StatementType::COMMENT_ON,
+              BoundStatementResult::createSingleStringColumnResult()},
+          tableID(tableID), tableName(tableName), comment(comment) {}
```

Comment:
better to use std::move() when initializing tableName, comment to avoid additional copy.
`tableName(std::move(tableName)), comment(std::move(comment))`